src/utiles/appx.ts

```diff
@@ -7,7 +7,43 @@ const APPX_BASE_API = process.env.APPX_BASE_API;
 const LOCAL_CMS_PROVIDER = process.env.LOCAL_CMS_PROVIDER;
 
 export async function getPurchases(email: string) {
-  const courses = await db.course.findMany({});
+  const _courses = await db.course.findMany({
```

Comment:
Does this assume that videos will only be at a `depth` of `1` ? Not a problem right now but could be in the future

Reply:
Yeah, it could be a problem, let me check how to fix it.